### PR TITLE
Refactor Awesome books app with ES6, modules, arrow functions, and Luxon library

### PR DESCRIPTION
**Pull request by ahmidou09**
**This pull request includes a major refactoring of the Awesome books app to adhere to the project requirements. The following changes have been made:**

- Re-created the Awesome books app using ES6 syntax for improved readability and modern JavaScript features.

- Divided the code into modules to promote modularity and maintainability.

- Imported modules into index.js for a more modular and structured approach.

- Ensured that all functionalities of the app are maintained without introducing any errors during the refactoring process.

- Refactored functions to arrow functions to leverage the concise syntax and lexical scoping benefits.

- Utilized let and const in the correct way to improve code clarity and prevent accidental reassignment.

- Integrated the Luxon library for handling dates and times effectively.

- Downloaded the Luxon library into the project using npm.

- Imported the Luxon library as a module to leverage its functionalities.

- Implemented the feature to display the current date below the navigation bar using the Luxon library.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import BookList from './modules/bookList.js';
+
+const book = new BookList();
+
+console.log(book);
```
Comment (usorfaitheloho) [inline index.js]: Kindly remove `console.log` from your js files after the implementation, as it is not a javascript best practice 😄

File: modules/bookList.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    });
+  };
+
+   displayCurrentDate = () => {
+     const header = document.querySelector('.header');
+     const currentDate = DateTime.now().toLocaleString(DateTime.DATETIME_MED);
+     const htmlDate = `<div class="currentDate">${currentDate}</div>`;
+     header.insertAdjacentHTML('afterend', htmlDate);
+   }
+}
```
Comment (usorfaitheloho) [inline modules/bookList.js:88]: According to the project requirement, you are to split your code into modules, however, you have just one module for your project. Kindly use at least two modules to show that you fully grasp it's usage 

image

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
> Hi @ahmidou09  ,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

  [ ](https://camo.githubusercontent.com/679757134a8fee76cfe1941f9f3ecb0e0cf56dd02da4498268ce31f0dacdfca3/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)
> 
> ## Highlights
> * Used Javascript modules ✔️
> * Used Javascript ES6 syntax ✔️
> * Used GitHub flow ✔️
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by addod19** [comment]
### Hi @ahmidou09, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that is invalid because linters are not set up correctly for this PR

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by Strangeal — APPROVED** [review]
Hi @ahmidou09, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Alt Text

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**